### PR TITLE
All IsoTorrents and TorrentProjects fix (pop-up)

### PR DESCRIPTION
**Pull request by wout-codes**
#73507, #73508, #73509, #73510, #73511, #73512, #73513, #73514, #73515, #73517, #73518, #73519, #73520, #73521, #73522
Only Pop-ups were fixed.

### CODE REVIEW

**Review by zloyden — APPROVED** [review]

**Comment by wout-codes** [comment]
There are ads left

**Comment by zloyden** [comment]
> There are ads left

Ok, can you fix it?

**Comment by wout-codes** [comment]
> > There are ads left
> 
> Ok, can you fix it?
I will try wait, I am not good with advanced syntax so yeah.

**Comment by zloyden** [comment]
@wout-codes try these rules
```
##iframe[src^="/nordv/"]
##.banner-wrp
##.post-sidebar
```

**Comment by wout-codes** [comment]
> @wout-codes try these rules
> 
> ```
> ##iframe[src^="/nordv/"]
> ##.banner-wrp
> ##.post-sidebar
> ```

Works for IsoHunt Torrents but not for TorrentProject

**Comment by zloyden** [comment]
> > @wout-codes try these rules
> > ```
> > ##iframe[src^="/nordv/"]
> > ##.banner-wrp
> > ##.post-sidebar
> > ```
> 
> Works for IsoHunt Torrents but not for TorrentProject

It's for `isohunt`. You can make similar rules for `torrentproject`.

File: EnglishFilter/sections/general_extensions.txt [hunk 1]
```diff
@@ -238,6 +238,12 @@ kickasstracker.*,katfreak.*,kickasshydra.*,katbay.*,kickasst.*,kkickass.*,kattra
 torrentproject2.*,torrentproject2.com,torrentproject2.net,torrentproject2.org#%#//scriptlet("prevent-window-open")
 !
 !
+! IsoHunts
+isohuntz.*,isohunt.*,isohunts.*,isohuntx.*,isohunthydra.*,isohunters.*,isohunting.*,myisohunt.*,isohuntz.net,isohunts.net,isohunt.soy,isohuntx.com,isohunthydra.com,isohunt.io,isohunters.org,isohunt.bz,isohunting.com,isohunting.com,isohunt.how,myisohunt.com,isohunt.lol##iframe[src^="/nordv/"]
+isohuntz.*,isohunt.*,isohunts.*,isohuntx.*,isohunthydra.*,isohunters.*,isohunting.*,myisohunt.*,isohuntz.net,isohunts.net,isohunt.soy,isohuntx.com,isohunthydra.com,isohunt.io,isohunters.org,isohunt.bz,isohunting.com,isohunting.com,isohunt.how,myisohunt.com,isohunt.lol##.banner-wrp
+isohuntz.*,isohunt.*,isohunts.*,isohuntx.*,isohunthydra.*,isohunters.*,isohunting.*,myisohunt.*,isohuntz.net,isohunts.net,isohunt.soy,isohuntx.com,isohunthydra.com,isohunt.io,isohunters.org,isohunt.bz,isohunting.com,isohunting.com,isohunt.how,myisohunt.com,isohunt.lol##.post-sidebar
+!
+!
```
Comment (zloyden) [inline EnglishFilter/sections/general_extensions.txt:246]: Wrong section. It's for `specific.txt`

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

**Comment by zloyden** [comment]
Well done:)

**Pull request merged**